### ISSUE TITLE
N900 audio blobs

### ISSUE THREAD

**Issue by jonwil** [maemo-leste/bugtracker#155]
If we want to get audio for Leste on the N900 that is as good as that in Fremantle (call audio in particular), someone needs to finish reverse engineering the PulseAudio blobs for the N900 (with all the special algorithms to make voice calls sound good)
Latest RE work is here
https://github.com/community-ssu/pulseaudio-nokia/commits/master
We will also need https://github.com/community-ssu/cmtspeech

**Comment by MerlijnWajer** [comment]
By the way, I am not sure -where- I documented this, but there are also some modules in newer pulseaudio that might do already what we want, so with a bit of luck we might not need to RE the modules fully. It might be in the IRC logs.

**Comment by robiwan1** [comment]
As an update, pulseaudio-module-cmtspeech is in the repo since 27 Jul 2021...

**Comment by MerlijnWajer** [comment]
Right, I've packaged them, but I haven't been able to fully test them yet. Help is very welcome. It's a start at least. :)
